metrics/core/metrics.go

```diff
@@ -71,6 +72,9 @@ var LabeledMetrics = []Metric{
 	MetricFilesystemAvailable,
 	MetricFilesystemInodes,
 	MetricFilesystemInodesFree,
+	MetricAcceleratorMemoryCapacity,
+	MetricAcceleratorMemoryUsed,
+	MetricAcceleratorUtility,
```

Comment:
Instead of Utility use DutyCycle. They are different things.

Reply:
Will do.